docs/en/deployment.md

```diff
@@ -1,14 +1,13 @@
 Before you launch the solution, review the architecture, supported regions, and other considerations discussed in this guide. Follow the step-by-step instructions in this section to configure and deploy the solution into your account.
 
 
-**Time to deploy**: Approximately 10 minutes(all parts)
+**Time to deploy**: Approximately 10 minutes
 
 !!! notice "Note"
 
     Before deploying this solution, we recommend you [create a billing alarm to monitor your estimated AWS charges](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/monitor_estimated_charges_with_cloudwatch.html). 
 
-The user or IAM role to perform the deployment must have at least [permissions](./workshop/a-molecular-unfolding/permissions.json). If you use the permissions defined in this file to deploy this solution, your CloudFormation stack name should start with `QCEDD`, e.g `QCEDDStack` .
-
+The user or IAM role to perform the deployment must have at least permissions defined in [this file](https://awslabs.github.io/quantum-computing-exploration-for-drug-discovery-on-aws/en/workshop/a-molecular-unfolding/permissions.json). If you use the permissions defined in this file to deploy this solution, your CloudFormation stack name should start with `QCEDD`, for example, `QCEDDStack`.
```

Comment:
any reason hardcode the url?

Reply:
because we need to accommodate global release